app/helpers/i18n_helper.rb

```diff
@@ -0,0 +1,17 @@
+module I18nHelper
+
+  private
```

Comment:
This got merged in an earlier PR already, but I would be happy to remove it in this PR.